src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,282 @@
+
+There are several advanced configuration options available for the `nr-k8s-otel-collector` Helm chart. These options allow you to customize the behavior of the OpenTelemetry Collector to better suit your needs. The chart supports both `DaemonSet` and `Deployment` collectors, allowing you to choose the best fit for your use case. For more information on installing the New Relic OpenTelemetry Collector in Kubernetes, refer to the [installation guide](/docs/kubernetes-pixie/k8s-otel/install).
+
+This document provides an overview of some of the key advanced configuration options you can use.
+
+
+## Enable GKE Autopilot or Red Hat OpenShift Compatibility [#enable-provider]
+
+
+To ensure compatibility with specific Kubernetes environments, you can enable provider-specific configurations. This setting ensures compatibility and proper functionality of the OpenTelemetry Collectors by adapting to the specific constraints of these environments. You can enable this option in your [`values.yaml`](/docs/kubernetes-pixie/k8s-otel/install/#config-params) file:
+
+```yaml
+  provider: "GKE_AUTOPILOT" # Or "OPEN_SHIFT" if applicable
+```
+
+
+## Enable Low Data Mode [#low-data]
+
+
+The `LowDataMode` option is enabled by default to ingest only the metrics required by our Kubernetes UIs. This mode reduces the amount of data collected, focusing on essential metrics for Kubernetes monitoring. To fetch additional metrics, you can add new pipelines and configuring the appropriate receivers and processors in your [`values.yaml`](/docs/kubernetes-pixie/k8s-otel/install/#config-params) file using the `extraConfig` section.
+
+The following is an example of how to add the `cadvisor_version_info` metric to a new pipeline. You can reuse existing receivers or define your own. Processors are added to filter specific metrics and enrich them with Kubernetes attributes.
+
+```yaml
+  extraConfig:
+    receivers:
+    processors:
+      filter/keep_cadvisor_version_info:
+        metrics:
+            metric:
+              - name != "cadvisor_version_info" # Exclude all metrics except cadvisor_version_info
+    exporters:
+    connectors:
+    pipelines:
+      metrics/additional_metrics:
+        receivers:
+          - prometheus # This references the prometheus receiver defined above
+        processors:
+          - filter/keep_cadvisor_version_info
+          - resource # Essential for basic resource attributes
+          - k8sattributes/ksm # Essential for Kubernetes metadata enrichment
+          - cumulativetodelta # Converts cumulative metrics to delta
+          - batch # For efficient data sending
+        exporters:
+          - otlphttp/newrelic
+```
+
+For a comprehensive list of available receivers, processors, exporters, and pipelines that you can reuse in your configurations, refer to the [New Relic Helm Charts repository.](/docs/kubernetes-pixie/k8s-otel/intro/#otel-components)
```

Comment:
The link is correct but it redirects to our public docs not to the NR Helm chart repository